lib/js/src/manager/screen/_SoftButtonManagerBase.js

```diff
@@ -60,6 +60,12 @@ class _SoftButtonManagerBase extends _SubManagerBase {
         this._currentMainField1 = null;
         this._currentHmiLevel = null;
         this._batchUpdates = false; // whether to wait on sending the updates
+        this._displayCapabilities = null;
+        if (lifecycleManager.getSystemCapabilityManager() !== null) {
+            this._displayCapabilities = lifecycleManager.getSystemCapabilityManager().getDisplayCapabilities();
+        }
+        this._isDynamicGraphicSupported = (this._displayCapabilities !== null && this._displayCapabilities.getGraphicSupported() !== null) ? this._displayCapabilities.getGraphicSupported() : true;
```

Comment:
```suggestion
        let displayCapabilities = null;
        if (lifecycleManager.getSystemCapabilityManager() !== null) {
            displayCapabilities = lifecycleManager.getSystemCapabilityManager().getDisplayCapabilities();
        }
        this._isDynamicGraphicSupported = (displayCapabilities !== null && displayCapabilities.getGraphicSupported() !== null) ? displayCapabilities.getGraphicSupported() : true;
```
We don't use the display capabilities elsewhere in the operation so we don't need to save it. This also aligns better with the Java implementation.